src/language/en.tsx

```diff
@@ -49,9 +49,9 @@ const en = {
     activityWeekHistoryTitleDay: 'Today\'s history',
     activityWeekHistoryTitleWeek: 'Activity history',
 
-    informationCardActivityscreen: 'Activityscreen',
-    informationCardValuesscreen: 'Valuesscreen',
-    informationCardCalenderscreen: 'Calenderscreen',
+    informationCardActivityscreen: 'Activityinfo',
+    informationCardValuesscreen: 'Valuesinfo',
+    informationCardCalenderscreen: 'Calenderinfo',
```

Comment:
I think the titles needs to be changed a bit.
```suggestion
    informationCardActivityscreen: 'Activities',
    informationCardValuesscreen: 'Values',
    informationCardCalenderscreen: 'Calendar',
```